group.go

```diff
@@ -4,14 +4,31 @@ import (
 	"github.com/shurcooL/graphql"
 )
 
+type GroupCreateInput struct {
+	Description string            `json:"description,omitempty"`
+	Members     []MemberInput     `json:"members,omitempty"`
+	Name        string            `json:"name,omitempty"`
+	Parent      IdentifierInput   `json:"parent,omitempty"`
+	Teams       []IdentifierInput `json:"teams,omitempty"`
+}
+
+type GroupDeleteInput struct {
```

Comment:
this struct is not used anywhere - lets delete it - probably was just leftover from noodling the code